File: app/src/main/java/com/lucas/hometask/Model/Casa.java
package com.lucas.hometask.model;

import java.util.ArrayList;

public class Casa {
    private int id;
    private String nome;
    private ArrayList<Usuario> moradores;
    private ArrayList<Tarefa> tarefas;

    public Casa(String nome) {
        this.nome = nome;
    }

    public Casa(int id, String nome) {
        this.id = id;
        this.nome = nome;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public ArrayList<Usuario> getMoradores() {
        return moradores;
    }

    public void setMoradores(ArrayList<Usuario> moradores) {
        this.moradores = moradores;
    }
}

File: app/src/main/java/com/lucas/hometask/model/Casa.kt
package com.lucas.hometask.model

import java.util.ArrayList

class Casa {
    var id: String? = null
    var nome: String? = null
    var imagem: String? = null
    var moradores: ArrayList<Usuario>? = null
    private val tarefas: ArrayList<Tarefa>? = null
    private val regras: ArrayList<Regra>? = null


    constructor(nome: String) {
        this.nome = nome
    }

    constructor()

    constructor(id: String, nome: String) {
        this.id = id
        this.nome = nome
    }

    fun addMorador(morador: Usuario) {
        if (moradores == null) moradores = ArrayList()
        moradores!!.add(morador)
    }
}
